src/Eccube/Entity/Order.php

```diff
@@ -47,6 +47,14 @@ class Order extends \Eccube\Entity\AbstractEntity implements PurchaseInterface,
      */
     private $errors = [];
 
+    /**
+     * Clone method
+     */
+    public function __clone()
+    {
+        $this->id = null;
```

Comment:
clone()を定義している理由を教えてください。

Reply:
Because we use clone Order but we do not implement clone function. If you do not do this, clone objects and entities will share UnitOfWork. It is not good if we manipulate the clone object.